tests/cases/conformance/jsdoc/jsdocTypeReferenceToMergedClass.ts

```diff
@@ -3,14 +3,12 @@
 // @allowJs: true
 // @checkJs: true
 
-// @Filename: test.js
```

Comment:
this is now a single-file repro. It is still written in such a way that `function wp` is checked *after* `p.isServiceProject`.